include/cloudbreak.bash

```diff
@@ -1,9 +1,12 @@
 
 cloudbreak-config() {
-  : ${BRIDGE_IP:=$(docker run --rm --name=cbreak_cbd_bridgeip --label cbreak.sidekick=true alpine sh -c 'ip ro | grep default | cut -d" " -f 3')}
-  env-import PRIVATE_IP $BRIDGE_IP
-  env-import DOCKER_MACHINE ""
-  compose-config
+  if is_macos; then
+    : ${BRIDGE_ADDRESS:=host.docker.internal}
+    env-import PRIVATE_IP 127.0.0.1
```

Comment:
we don' need the PRIVATE_IP anymore, or do we?

Reply:
we are using only in uaa.yaml, I will try to remove and see what happens

Reply:
it looks fine too after I removed PRIVATE_IP

Reply:
I think Periscope is the one that uses it. Worth trying if it still work in case you run it from idea.